lib/project_types/theme/messages/messages.rb

```diff
@@ -2,12 +2,47 @@
 module Theme
   module Messages
     MESSAGES = {
-      ensure_themekit_installed: {
-        downloading: "Downloading Themekit %s",
-        failed: "Download failed",
-        successful: "Themekit installed successfully",
-        unsuccessful: "Unable to verify download digest",
-        verifying: "Verifying download...",
+      theme: {
+        create: {
+          creating_theme: "Creating theme %s",
+          checking_themekit: "Verifying Theme Kit",
+          failed: "Theme could not be created",
+          help: <<~HELP,
+            {{command:%s create theme}}: Creates a theme.
+              Usage: {{command:%s create theme}}
+              Options:
+                {{command:--store=MYSHOPIFYDOMAIN}} Store URL. Must be an existing store.
+                {{command:--password=PASSWORD}} Private app password. App must have Read and Write Theme access.
+                {{command:--name=NAME}} Theme name. Any string.
+          HELP
+          info: {
+            created: "{{green:%s}} was created for {{underline:%s}} in {{green:%s}}",
+          },
+        },
+        forms: {
+          create: {
+            ask_password: "Password:",
+            ask_store: "Store domain:",
+            ask_title: "Title:",
+            errors: "%s cannot be empty",
+            private_app: <<~APP,
+              To create a new theme, we need to connect with a private app. Visit {{underline:%s/admin/apps/private}} to fetch the password.
```

Comment:
```suggestion
              To create a new theme, Theme Kit needs to connect to a private app installed on your store. Visit {{underline:%s/admin/apps/private}} to create a new API key and password, or retrieve an existing password.
```

In general we (Shopify) try not to call ourselves "we" if we can help it. 

From a UX perspective, are people using Theme Kit _through_ Shopify App CLI likely to be aware that they're using Theme Kit? Or does Theme Kit just manage things under the hood without its name being exposed to end users?

If it's the former, it should be `Theme Kit needs to connect`. If it's the latter, it should be `Shopify App CLI needs to connect`.